rosdep/base.yaml

```
@@ -3456,6 +3456,27 @@ libflann-dev:
   openembedded: [libflann@meta-ros-common]
   rhel: [flann-devel]
   ubuntu: [libflann-dev]
+libflatbuffers:
```

To match the Ubuntu name, I think this should be:

```suggestion
libflatbuffers-dev:
```